src/main/java/org/primefaces/component/selectcheckboxmenu/SelectCheckboxMenuBase.java

```diff
@@ -202,4 +203,12 @@ public void setDynamic(boolean dynamic) {
     public String resolveWidgetVar() {
         return ComponentUtils.resolveWidgetVar(getFacesContext(), this);
     }
+
+    public String getLabelSeparator() {
+        return (String) getStateHelper().eval(PropertyKeys.labelSeparator, ",");
+    }
+
+    public void setLabelSeparator(String labelSeparator) {
+        getStateHelper().put(PropertyKeys.labelSeparator, labelSeparator);
+    }
```

Comment:
I try and move all setters and getters ABOVE the resolveWidgetVar method.  The resolveWidgetVar is usually the bottom most method.